### ISSUE TITLE
Ender Chest = Invalid Inventory ? Should be good. (Suggestion)

### ISSUE THREAD

**Issue by Knito58** [McJty/RFToolsControl#83]
The processor does not recognize MC Ender Chest as valid inventory. As soon as I put a normal chest in its place, it works. Would be cool. :)

**Comment by McJty** [comment]
That's because it's not possible.  The enderchest has another inventory depending on player and cannot be accessed with machines. A hopper does not work either

**Comment by Knito58** [comment]
Ok. I remember I already heared this....

What about an ender chest interface (ECI) that would contact a normal chest at a given position in the world, say at 0, -1, 0? And sync that with the player's ender chests? And the interface would in real only go to the hidden chest of the player. --- Ok I'm starting to babble silly things now.

My "solution" has a real multiplayer problem. 

If I was content with what I've got I would never have installed any mod.


Thanks for modding and taking care of your followers.

Greetings, Konitor


If your mod is dull, then nobody cares. If its already good, then people have ideas to make it even better.
